sympy/integrals/transforms.py

```diff
@@ -1785,6 +1785,34 @@ def _laplace_rule_diff(f, t, s, doit=True, **hints):
     return False, (fn, p, c)
 
 
+def _laplace_rule_sdiff(f, t, s, doit=True, **hints):
+    """
+    """
+    fn, p, c = _laplace_ct(f)
+
+    if fn.is_Mul:
+        pfac = [fac for fac in Mul.make_args(fn) if fac.is_polynomial(t)]
+        if len(pfac)>0:
+            pex = prod(pfac)
+            pc = Poly(pex, t).all_coeffs()
+            N = len(pc)
+            if N>1:
+                debug('_laplace_apply_rules match:')
+                debug('      f, n: %s, %s'%(fn, pfac))
+                debug('      rule: frequency derivative')
+                rem = (fn/pex).simplify(doit=False)
+                r_, p_, c_ = LaplaceTransform(rem, t, s).doit()
+                if r_.has(LaplaceTransform):
+                    r = Add(*[ (-1)**n*pc[N-n-1]*Derivative(r_, s, n)
+                          for n in range(N) ]).simplify(doit=False)
```

Comment:
Again we should generally avoid calling `simplify` in library code. If there is some specific simplification that is wanted then that can be used (e.g. `expand_mul`, `cancel`, `together` etc).

Also it would be better to compute these derivatives in a loop because `Derivative(r, s, n)` essentially calls `diff` `n` times meaning that this loop uses a quadratic number of `diff` calls.

Reply:
Derivatives now done in loop, and simplify removed (that was an un-removed left-over from development)